docs/index.rst

```diff
@@ -26,6 +25,7 @@ xclim currently provides over 50 indices related to mean, minimum and maximum da
    sdba
    analogues
    api
+   modules
```

Comment:
Oh and this file doesn't exist, may be a missing `git add` ?

Reply:
This file is generated by sphinx-automodule. No need!

Reply:
Oh I see! But I see no menu entry between API and Contributing, is the generated file empty?

Reply:
Will verify!